models/sharky/solana/sharky_solana_events.sql

```diff
@@ -12,7 +12,7 @@
     )
 }}
 
-{%- set project_start_date = '2022-12-30' %} -- TODO change back to '2022-04-14'
+{%- set project_start_date = '2022-11-31' %} -- TODO change back to '2022-04-14'
```

Comment:
Model building success in under 1 minute for 2 month's data.
https://github.com/duneanalytics/spellbook/actions/runs/4056573085/jobs/6981237327#step:9:13

Reply:
Cool, I think it is time to pass the code review to @jeff-dude.

The building time is not stable though.